services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/docker_service_specs/_constants.py

```diff
@@ -0,0 +1,10 @@
+from typing import Final
+
+_NANO: Final[int] = int(1e9)
+
+
+DOCKER_CONTAINER_SPEC_RESTART_POLICY_DEFAULTS = {
+    "Condition": "on-failure",
+    "Delay": 100,
+    "MaxAttempts": 1 * _NANO,
```

Comment:
what means 1*10-9 of MaxAttempts? 0?
is that really what should be there?

Reply:
and what is the unit of Delay 100? ms? I am not so sure I understand the point of changing this. if these are milliseconds, I feel this a bit short.Why not use the default of 5 seconds? It would be a bit less agressive on the docker engine.

Reply:
Ohh dear... You are correct. This is bad.

Reply:
I've tried to make it a bit better. Let me see if it reads and makes sense to you now.
The unit of Delay I think it is nano seconds. Could not find it on the official docs. Previously we were assuming milliseconds.

Reply:
I've set it to 1 second now.

Reply:
how come you cannot test that on your local deploy?? and why change it from 5seconds.. is it a problem?